src/domain/app/i18n/en.json

```diff
@@ -1438,6 +1442,7 @@
     "permission": "permission",
     "place": "place",
     "platform": "platform",
+    "priceGroup": "price, group",
```

Comment:
```suggestion
    "priceGroup": "price group",
```
I suppose comma was not meant to be here?

Reply:
Changed to `customer group` to unify the terms